docs/team/kokrui.md

```diff
@@ -3,7 +3,44 @@
   title: "Wong Kok Rui's Project Portfolio Page"
 ---
 
-### Project: CampusConnect
+# Project: CampusConnect
 
 to be added soon
 
+# Summary of Contributions
+
+## Code Contributed
+
+to be added soon
+
+## Enhancements Implemented
+
+to be added soon
+
+## Contributions to the User Guide
+
+to be added soon
+
+## Contributions to the Developer Guide
+
+to be added soon
+
+## Contributions to Team-Based Tasks
+
+to be added soon
+
+## Review / Mentoring Contributions
+
+to be added soon
+
+## Contributions Beyond the Project Team
+
+to be added soon
+
+# Contributions to the Developer Guide (Extracts)
```

Comment:
Why don't you use ## here as well?

Reply:
I did it because based on the 2103 website

https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html#4-add-a-skeletal-ppp

<img width="554" alt="image" src="https://github.com/AY2324S1-CS2103T-T13-2/tp/assets/29654756/3c30f66d-3cf4-42d9-8ade-d08943522d7e">
<img width="596" alt="image" src="https://github.com/AY2324S1-CS2103T-T13-2/tp/assets/29654756/50d54b15-e9cf-4b7a-ace4-f598390d0557">

It seems like the (Extracts) sections are supposed to be distinct from Overview and Summary of Contributions. My header levels follow their bulletpoint indentation levels

Reply:
That said, happy to switch it up in a future milestone if we decide we want to deviate from this